src/views/FindOrder.vue

```diff
@@ -396,6 +396,25 @@ ion-modal {
   --border-radius: 8px;
 }
 
+main > div:hover {
+  cursor: pointer;
+  background: rgb(240, 240, 240);
+}
+
+ion-item, ion-card{
```

Comment:
we should not set their background to none, instead create a universal variable that can set the bg of various elements from one selector.

Reply:
As per our discussion, I have removed hover effect changes and only added cursor pointer to order div.